fuse_models/test/test_unicycle_2d_state_cost_function.cpp

```diff
@@ -99,7 +99,17 @@ TEST(CostFunction, evaluateCostFunction)
 
   // Check jacobians are correct using a gradient checker
   ceres::NumericDiffOptions numeric_diff_options;
-  ceres::GradientChecker gradient_checker(&cost_function, nullptr, numeric_diff_options);
```

Comment:
This looks like a change from another MR

If possible, get rid of this change here